### ISSUE TITLE
search overlay webhook yields JS console errors

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#5522]
the error comes from this [line](https://github.com/galaxyproject/galaxy/blob/b4cdce925bf485ecdb24868bacdcffe2f4e7a717/config/plugins/webhooks/demo/search/script.js#L511) and it occurs 3 times for me using any search term

ping @anuprulez 

```
VM2870:511 Uncaught ReferenceError: item is not defined
    at eval (eval at globalEval (jquery.js?299b:9814), <anonymous>:511:21)
    at Function._.each._.forEach (underscore.js?bb1e:153)
    at child.makeToolSection (eval at globalEval (jquery.js?299b:9814), <anonymous>:475:15)
    at child.makeSection (eval at globalEval (jquery.js?299b:9814), <anonymous>:411:26)
    at child.makeAllSection (eval at globalEval (jquery.js?299b:9814), <anonymous>:445:26)
    at child.refreshView (eval at globalEval (jquery.js?299b:9814), <anonymous>:388:22)
    at child.initialize (eval at globalEval (jquery.js?299b:9814), <anonymous>:363:22)
    at child.Backbone.View (backbone.js?5b87:1224)
    at new child (backbone.js?5b87:1884)
    at Object.eval [as success] (eval at globalEval (jquery.js?299b:9814), <anonymous>:198:30)
```

**Comment by anuprulez** [comment]
I will look into it. Thanks! @martenson

**Comment by martenson** [comment]
thank you!

**Comment by hexylena** [comment]
Fixed in #5531
